scipy/sparse/linalg/tests/test_propack.py

```diff
@@ -111,10 +111,13 @@ def test_svdp(ctor, precision, irl, which):
 @pytest.mark.parametrize('precision', _dtype_testing)
 @pytest.mark.parametrize('irl', (False, True))
 def test_examples(precision, irl):
+    # Note: atol for complex8 bumped from 1e-4 to 1e-3 because of test failures
+    # with BLIS, Netlib, and MKL+AVX512 - see
+    # https://github.com/conda-forge/scipy-feedstock/pull/198#issuecomment-999180432
     atol = {
-        'single': 1e-4,
+        'single': 1.2e-4,
```

Comment:
@mckib2 sets this to `1e-3` in #15249 - which one should it be?

In [adding](https://github.com/conda-forge/scipy-feedstock/pull/198/commits/22b4dbdbec6c65ef1c2061e99a9c768390bafaa8) this PR to https://github.com/conda-forge/scipy-feedstock/pull/198, I've gone with the tighter bound for now.

Reply:
It's a little arbitrary - we usually set it with some small-ish margin on top of the failure we observe. So whichever one gets merged first I guess, unless `1.2` is not enough:)

Reply:
Yes, please try `1.2` -- I didn't realize these were being tracked here